auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -37,11 +48,97 @@
                 <div>{{ displayStatus(item.status) }}</div>
               </template>
 
-              <!-- Displaying Formatted Date -->
+              <!-- Displaying Formatted Submission Date -->
               <template #[`item.submissionDate`]="{ item }">
                 <div>{{ formatDate(item.submissionDate) }}</div>
               </template>
 
+              <!-- Displaying Formatted Future Effective Date and Tooltips-->
+              <template #[`item.futureEffectiveDate`]="{ item }">
+                <div>
+                  <v-tooltip bottom>
+                    <template #activator="{ on, attrs }">
+                      <span
+                        v-bind="attrs"
+                        style="border-bottom: 1px dotted"
+                        v-on="on"
+                      >
+                        {{ formatDate(item.futureEffectiveDate) }}
+                      </span>
+                    </template>
+                    <div class="bottom-tooltip">
+                      <span>{{ formatDate(item.futureEffectiveDate) }} at {{ formatTime(item.futureEffectiveDate) }} Pacific time</span>
```

Comment:
I try to show the little arrow on top of the tooltip, but I'm not sure why this class="bottom-tooltip" didn't work
![image](https://github.com/user-attachments/assets/9a72c066-071a-44d2-be67-7557143a1881)

(it's the same style used by the IconTooltip component)

Reply:
Try without `::v-deep`.

Reply:
We need to use v-deep when the HTML is rendered by a Vuetify component.

Here, it's "static" HTML in the template.

Reply:
I tried without ::v-deep, but it didn't work either

Reply:
It seems I couldn't change the styles of the tooltip

Reply:
I see that `:before` isn't in the DOM. Let me see if I can figure it out.

![image](https://github.com/user-attachments/assets/764cbbc0-b1f6-4ee2-a857-e0f819bd39fc)

Reply:
Found it. It's because your CSS is nested in the form. Move it outside and the v-deep will work.

PS You'll have to fix the colour of the pointer.

Reply:
Thanks a lot! Fixed it and fixed the color:
![image](https://github.com/user-attachments/assets/313c0d8d-d12c-40b9-854a-385e9db23d00)

Reply:
Looks great! 🎉